src/Layers/FeatureLayer/FeatureLayer.js

```diff
@@ -90,8 +103,20 @@ EsriLeaflet.Layers.FeatureLayer = EsriLeaflet.Layers.FeatureManager.extend({
         return;
       }
 
+      // polylines and polygons
       if (layer && layer.setLatLngs) {
-        this._updateLayerGeometry(layer, geojson);
+        this._updateLayer(layer, geojson);
+        // to do: make sure appropriate style is set
+        // this.resetStyle();  ?
+        return;
+      }
+
+      // points
+      if (layer && layer.setLatLng) {
+        this._updateLayer(layer, geojson);
+        var iconTest = this.options.pointToLayer(geojson, L.latLng(geojson.geometry.coordinates[1], geojson.geometry.coordinates[0]));
+        var updatedIcon = iconTest.options.icon;
+        layer.setIcon(updatedIcon);
```

Comment:
The iconTest name is a little confusing here, are you expecting that there might not be an icon or a pointToLayer function?  If so, what happens in layer.setIcon if updateIcon is null?  Does it break or just use the default leaflet icon?